algorithms/ar-isrt/typescript/insertion_sort.ts

```diff
@@ -0,0 +1,24 @@
+const swap = (arr: Array<number>, a: number, b: number) => {
```

Comment:
Hi @DanielRomeo 
Remove this file, this is not acceptable,

Reply:
hello @Danushka96  what did i do wrong?

Reply:
@DanielRomeo This PR is about stack right? why there's an insertion sort in the commits?

Reply:
oh. I have no idea, my insertion sort pull request was accepted a few hours ago, i don't know why my commit that got added again to this PR. Now I'm confused, what exactly should I do? should I delete this PR entirely and create a new one? or?

Reply:
@Danushka96

Reply:
Yes, let's make a clear PR,